scss/components/_accordion-menu.scss

```diff
@@ -51,7 +51,7 @@ $accordionmenu-submenu-toggle-height: $accordionmenu-submenu-toggle-width !defau
 $accordionmenu-arrow-size: 6px !default;
 
 @mixin zf-accordion-menu-left-right-arrows {
-  .is-accordion-submenu-parent:not(.has-submenu-toggle) > a {
+  .is-accordion-submenu-parent:not(.has-submenu-toggle) > a {  // sass-lint:disable-line force-element-nesting force-pseudo-nesting
```

Comment:
I meant: disabling it for the entire project in `.sass-lint.yml`.
You can squash commits after this.

Reply:
> I meant: disabling it for the entire project in .sass-lint.yml.
> You can squash commits after this.

Done